packages/viewer/src/lib/interactions/flyToPartKeyInteraction.ts

```diff
@@ -15,7 +15,7 @@ export class FlyToPartKeyInteraction
   ) {}
 
   public predicate(e: TapEventDetails): boolean {
-    return !!e.altKey;
+    return !!e.altKey && !e.shiftKey;
```

Comment:
`e.altKey && !e.shiftKey`?

Reply:
Updated in https://github.com/Vertexvis/vertex-web-sdk/pull/313/commits/e3727dc4690dd42f62f6247a6e3ba39713e205b6. Copied over the `!!` syntax from the existing fly to part interaction, but it does look like that's unnecessary now